### DISCUSSION TITLE
Gogrepos script downloads all language version.

### DISCUSSION THREAD

**Discussion by MrYTachi** [retronas/discussions#14] [Ideas]
Well this is probably more like a suggestion. I am having a problem with the script that it keeps downloading every version of the game in my gog library like baldur's gate 2  complete edition downloaded for me german, polish, english, french, russian editions of the game. Would be nice if there was a way to limit the script to download just one language version of the game or which ever any other person need . So that you can limit the wasted space and extra work to go back an remove them manually.
![Näyttökuva 2023-08-09 234906](https://github.com/danmons/retronas/assets/141875298/aa428dac-84e2-40a1-99f1-25285fde91c6)

**Comment by sairuk** [comment]
feature request so converted to discussion

we could probably provide the ability to set `-lang` which seems to be supported in the main script

e.g https://github.com/eddie3/gogrepo/blob/2cc5da4b00d69c8ab4d2482a5544398a63c35198/gogrepo.py#L457

**Reply by sairuk** [reply]
i took a look at the source for gogrepo and it should only be downloading the english version (not all version) if `-lang` is not specified, so the fact that it is currently downloading all may be down to a change on the gog end

**Reply by MrYTachi** [reply]
Maybe... I forgot to mention I used the gogrepo script from within the retronas.sh tools, It has been so long since I last coded so I got nowhere with my search. I think -lang is not specified when used from within retronas script but that is just my guess for now.
